setup.py

```diff
@@ -13,7 +13,6 @@
     install_requires=[
         # some "concrete" dependencies in requirements.txt
         'Flask>=1.1',
-        'Flask_SQLAlchemy',
```

Comment:
You'll need to add a dependency for sqlalchemy

Reply:
Thanks. We had one below. I've removed the py3 constraint and moved it up.